src/components/callout/callout.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+
+interface CalloutProps {
+  illustration?: React.ReactNode;
+  children: ReactNode | ReactNode[];
+}
+
+const CalloutLayout = styled.div`
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 16px 20px;
+  color: #3e61ee;
+  border: 2px solid currentColor;
+  box-sizing: border-box;
+
+  ${up('sm')} {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const CallOutText = styled.div`
+  ${up('sm')} {
+    flex: 1 1 auto;
+  }
+`;
+const CallOutIcon = styled.div`
+  float: left;
+  width: 48px;
+  height: 48px;
+  padding-right: 0.5em;
+
+  ${up('sm')} {
+    padding-right: 0;
+    padding-left: 0.5em;
+    width: 72px;
+    height: 72px;
+    flex: 0 0 72px;
+    order: 1;
+  }
+`;
+
+/**
+ - Use an avatar for attributing actions or content to specific users.
+ - The user's name should always be present when using Avatar – either printed beside the avatar or in a tooltip.
+ **/
```

Comment:
Is this related to this component? I don't understand...

Reply:
I tinkered around with the comments, and I copied this piece of text from some working solution. I lost my actual comment. This text should be recognized by Storybook as the description but it's not. I have now placed the correct description in there again and ignore the fact it's not displayed for now.